notify/templates_test.go

```diff
@@ -102,7 +102,7 @@ func TestTemplateSimple(t *testing.T) {
 				Kind: ExecutionError,
 				Error: template.ExecError{
 					Name: "slack.title",
-					Err:  errors.New(`template: slack.title:1:38: executing "slack.title" at <{{template "missing" .}}>: template "missing" not defined`),
```

Comment:
The `slack.title` in the error message came from `FromGlobs`?

Reply:
No, the extra `slack.title` in the error came from the `TestTemplate` code, it was parsing slightly differently than "real" path it was trying to mimic. Note the removed `_, err = newTextTmpl.New(c.Name).Parse(c.Template)`.

Now that we can parse templates directly, we use the same methods so it should always be inline with the actual error that would happen when ran via notifications.